### ISSUE TITLE
Unsure of ```enforceWarningList=True``` behavior

### ISSUE THREAD

**Issue by ninSmith** [MISP/PyMISP#199]
I'm using pymisp to pull misp event attributes with:

```
mispApi.search(last='1d', enforceWarninglist=True)
```

However, it seems that using ```enforceWarninglist=True``` I got no attribute at all.   
I thought only attributes in warninglist were filtered out when using ```enforceWarninglist=True``` , am I missing something ?   

Below what I tried and response I got:

```misp.search(last='1d', enforceWarninglist=False``` gives me:

```
{
	'response': [{
		'Event': {
			'Attribute': [{
					'ShadowAttribute': [],
					'category': 'Payload delivery',
					'comment': '',
					'deleted': False,
					'disable_correlation': False,
					'distribution': '5',
					'event_id': '6757',
					'id': '510751',
					'object_id': '0',
					'object_relation': None,
					'sharing_group_id': '0',
					'timestamp': '1519744484',
					'to_ids': True,
					'type': 'domain',
					'uuid': '5a9575e4-07ac-4c6c-ac3b-fe97ac16600a',
					'value': 'foo.bar'
				},
				{
					'ShadowAttribute': [],
					'category': 'Payload delivery',
					'comment': '',
					'deleted': False,
					'disable_correlation': False,
					'distribution': '5',
					'event_id': '6757',
					'id': '510853',
					'object_id': '0',
					'object_relation': None,
					'sharing_group_id': '0',
					'timestamp': '1519745863',
					'to_ids': False,
					'type': 'ip-dst',
					'uuid': '5a957b47-c284-4df4-b3d4-01ecac16600a',
					'value': '8.8.8.8'
				}
			],
			'Galaxy': [],
			'Object': [],
			'Org': {
				'id': '1',
				'name': 'FOO',
				'uuid': '595664b3-ce7c-4709-8dee-ab89ac16600a'
			},
			'Orgc': {
				'id': '1',
				'name': 'FOO',
				'uuid': '595664b3-ce7c-4709-8dee-ab89ac16600a'
			},
			'RelatedEvent': [],
			'ShadowAttribute': [],
			'analysis': '0',
			'attribute_count': '2',
			'date': '2018-02-27',
			'disable_correlation': False,
			'distribution': '0',
			'event_creator_email': 'foo@bar.org',
			'id': '6757',
			'info': 'test',
			'locked': False,
			'org_id': '1',
			'orgc_id': '1',
			'proposal_email_lock': False,
			'publish_timestamp': '1519745867',
			'published': True,
			'sharing_group_id': '0',
			'threat_level_id': '1',
			'timestamp': '1519745863',
			'uuid': '5a9575cc-9d6c-4043-9dcd-fe98ac16600a'
		}
	}]
}
```

And ```misp.search(last='1d', enforceWarninglist=True``` gives me:

```
{
	'response': [{
		              {'Event': {'Attribute': [],
                         'Galaxy': [],
                         'Object': [],
                         'Org': {'id': '1',
                                 'name': 'FOO',
                                 'uuid': '595664b3-ce7c-4709-8dee-ab89ac16600a'},
                         'Orgc': {'id': '1',
                                  'name': 'FOO',
                                  'uuid': '595664b3-ce7c-4709-8dee-ab89ac16600a'},
                         'RelatedEvent': [],
                         'ShadowAttribute': [],
                         'analysis': '0',
                         'attribute_count': '2',
                         'date': '2018-02-27',
                         'disable_correlation': False,
                         'distribution': '0',
                         'event_creator_email': 'foo@bar.org',
                         'id': '6757',
                         'info': 'test',
                         'locked': False,
                         'org_id': '1',
                         'orgc_id': '1',
                         'proposal_email_lock': False,
                         'publish_timestamp': '1519745867',
                         'published': True,
                         'sharing_group_id': '0',
                         'threat_level_id': '1',
                         'timestamp': '1519745863',
                         'uuid': '5a9575cc-9d6c-4043-9dcd-fe98ac16600a'}}
	}]
}
```

Of course, i made sure that ```foo.bar``` and ```8.8.8.8``` are not in any warninglist I use.    
I'm using pymisp 2.4.87.1

**Comment by Rafiot** [comment]
@iglocska I think this one is on you :p

**Comment by iglocska** [comment]
Tested on my end, works fine.

What I did:

Created 2 events

Event 1 (1 attribute):
  - 6.6.6.6

Event2 (2 attributes):
 - 6.6.6.6
 - 8.8.8.8

query `events/restSearch/json` with the following payload:
```
{
  	"value":"6.6.6.6"
}
```
returns both events, with all 3 attributes.

Repeating the query with the following payload:

```
{
	"enforceWarninglist":1,
  	"value":"6.6.6.6"
}
```

returns both events, but with each only having 1 attribute (8.8.8.8 was stripped).
